### ISSUE TITLE
Restoring from backup

### ISSUE THREAD

**Issue by sebastianmacarescu** [josegonzalez/python-github-backup#246]
Hello,

Is there any documented way to restore from a backup done with this project?

Thank you

**Comment by josegonzalez** [comment]
No, nothing documented.

**Comment by joaomas** [comment]
I also didn't see anything about restore, I don't think it has this functionality.

GitProtect.io has this feature but it is paid for more than 15 repositories.

**Comment by josegonzalez** [comment]
Yeah I originally wrote this as literally a backup utility with no thought in mind for restoring. I suppose its possible to restore most things, but nothing for that use case is covered here. Pull requests welcome!
